changelog.md

```diff
@@ -9,6 +9,8 @@ _Nothing yet._
 **Added:**
 - react-native-bpk-component-button: 4.2.16 => 4.3.0
   - New `iconAlignment` prop allows icons inside buttons to be leading or trailing.
+- bpk-react-utils:
```

Comment:
This includes `bpk-component-modal` too.

Also there needs to be an entry for `bpk-component-dialog`:
```
**Fixed:**
- bpk-component-dialog:
  - Setting `dissmissable` to false now correctly disables the close on `esc` key press
```

Reply:
Bearing in mind that `bpk-component-dialog` is still private and pre-1.0, do we still need to do this?

Reply:
Nope you are correct! apologies